descartes_core/src/cart_trajectory_pt.cpp

```diff
@@ -203,7 +211,78 @@ bool CartTrajectoryPt::getClosestJointPose(const std::vector<double> &seed_state
                                            const RobotModel &model,
                                            std::vector<double> &joint_pose) const
 {
-  NOT_IMPLEMENTED_ERR(false);
+  Eigen::Affine3d nominal_pose,candidate_pose;
```

Comment:
Agreed, the current approach may not be sufficient to address the more complex cases.  Issue #43 has been updated to include this observation.